aiocache/base.py

```diff
@@ -359,6 +359,31 @@ async def delete(self, key, namespace=None, _conn=None):
     async def _delete(self, key, _conn=None):
         raise NotImplementedError()
 
+    @API.register
+    @API.aiocache_enabled(fake_return=0)
+    @API.timeout
+    @API.plugins
+    async def multi_delete(self, keys, namespace=None, _conn=None):
+        """
+        Deletes multiple keys from the cache.
+
+        :param keys: list of str keys to be deleted
+        :param namespace: str alternative namespace to use
+        :param timeout: int or float in seconds specifying maximum timeout
+            for the operations to last
+        :returns: int number of deleted keys
+        :raises: :class:`asyncio.TimeoutError` if it lasts more than self.timeout
+        """
+        start = time.monotonic()
+        ns_keys = [self.build_key(key, namespace) for key in keys]
```

Comment:
I'm thinking we should make this immutable:
```suggestion
        ns_keys = tuple(self.build_key(key, namespace) for key in keys)
```